CIP-XXXX/README.md

```diff
@@ -0,0 +1,179 @@
+---
+CIP: ????
+Title: Royalty Datum Metadata Standard
```

Comment:
```suggestion
Title: Royalty Datum Metadata
```
I wanted to include @KtorZ @Ryun1 @SebastienGllmt in a discussion of whether we should use the term Standard here.  I would anticipate problems if so, because another metadata solution might also address royalties in the future and then we would have uncertainty over which one is "standard" while leaving that word out would force the unambiguous use of a merged CIP number.

(CIP27 wasn't so much a problem in this way because they used the commercial designation "CNFT" in addition to "Standard" in the merged CIP title.)

@SamDelaney since the term "standard" is used so many places in the document I would recommend just changing it in the header metadata & the PR title and leaving everything else as-is: to keep the tone & clarity of what you have already written, therefore leaving more detailed readers to make up their own mind over whether to consider this a "standard".